docs/data/pages.ts

```diff
@@ -432,16 +432,16 @@ const pages: MuiPage[] = [
             subheader: 'Common features',
             children: [
               { pathname: '/x/react-charts/axis' },
-              { pathname: '/x/react-charts/components', title: 'Custom components' },
+              { pathname: '/x/react-charts/components' },
```

Comment:
I thought the title for these pages came from the "title" property inside the MD file, which would make this title here unnecessary. But it seems I was wrong 🫠

Reply:
Yeah, I got it now, but it is not clear, since we are providing meta inside the md file itself, we could provide all the meta there instead and only have the structure here 😅